test/ctor-eval/gc-2.wast

```diff
@@ -0,0 +1,54 @@
+(module
```

Comment:
Could we make these `lit` tests? That would make review easier.

Reply:
This came up before in a previous PR. I think they could, yes, but it seems odd to make that transition now on what is possibly the very last of this long sequence of PRs :smile: